ckanext/qa/templates/activity_streams/activity_stream_items.html

```diff
@@ -0,0 +1,17 @@
+{% ckan_extends %}
+
+{% block activity_stream %}
```

Comment:
Because this is now moved to activity_stream, and the openness block is removed from the organization snippet - what happens now with the datasets that actually do have an organization? Will the openness info be displayed for those datasets?

Reply:
Yes. The openness snippet will be displayed on the bottom of activity stream for a data set. For the user the place remains the same.

Reply:
OK :+1: